repos.csv

```diff
@@ -47644,3 +47644,986 @@ scmHost,repoName,repoBranch,mavenTool,gradleTool,jdkTool,repoStyle,repoBuildActi
 ,zzz40500/Tinker_imitator,master,,,,,,TRUE,Build uses Gradle 2.x
 ,zzz40500/android-shapeLoadingView,master,,,,,,TRUE,Gradle wrapper 3.3 is not supported
 ,zzzzbw/Fame,master,,,,,,,
+",11ty/eleventy,main,,,,,,,"
+",1backend/1backend,master,,,,,,,"
+",1j01/jspaint,master,,,,,,,"
```

Comment:
We'll want to remove the wrapping quotes here.

Reply:
yep, removing…

Reply:
spotted that _almost_ right away